core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/validation/whitelist/ValidationWhiteListSearchRequestBuilder.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.core.validation.whitelist;
+
+import com.b2international.snowowl.core.ComponentIdentifier;
+import com.b2international.snowowl.core.ServiceProvider;
+import com.b2international.snowowl.core.request.SearchResourceRequest;
+import com.b2international.snowowl.core.request.SearchResourceRequestBuilder;
+import com.b2international.snowowl.core.request.SystemRequestBuilder;
+import com.b2international.snowowl.core.validation.whitelist.ValidationWhiteListSearchRequest.OptionKey;
+
+/**
+ * @since 6.1
+ */
+public final class ValidationWhiteListSearchRequestBuilder 
+	extends SearchResourceRequestBuilder<ValidationWhiteListSearchRequestBuilder, ServiceProvider, ValidationWhiteLists> 
+	implements SystemRequestBuilder<ValidationWhiteLists> {
+
+	ValidationWhiteListSearchRequestBuilder() {}
+	
+	public ValidationWhiteListSearchRequestBuilder filterByRuleId(String ruleId) {
+		return addOption(OptionKey.RULE_ID, ruleId);
+	}
+
+	public ValidationWhiteListSearchRequestBuilder filterByComponentIdentifier(ComponentIdentifier componentIdentifier) {
```

Comment:
Add methods to `filterByRuleIds(Iterable)` and `filterByComponentIdentifiers(Iterable)` to allow clients to filter by multiple rule and component identifier values.